drivers/hmis/app/graphql/types/forms/enums/data_collected_about.rb

```diff
@@ -10,12 +10,20 @@ module Types
   class Forms::Enums::DataCollectedAbout < Types::BaseEnum
     graphql_name 'DataCollectedAbout'
 
-    value 'ALL_CLIENTS'
-    value 'HOH_AND_ADULTS'
-    value 'HOH'
-    value 'ALL_VETERANS'
-    value 'ALL_CLIENTS_RECEIVING_SSVF_SERVICES'
-    value 'ALL_CLIENTS_RECEIVING_SSVF_FINANCIAL_ASSISTANCE'
-    value 'VETERAN_HOH'
+    [
+      'ALL_CLIENTS',
+      'HOH_AND_ADULTS',
+      'HOH',
+      'ALL_VETERANS',
+      'ALL_CLIENTS_RECEIVING_SSVF_SERVICES',
+      'ALL_CLIENTS_RECEIVING_SSVF_FINANCIAL_ASSISTANCE',
+      'VETERAN_HOH',
+    ].each do |val|
+      description = val.titleize.
+        gsub(/\bHoh\b/, 'HoH').
+        gsub(/\bSsvf\b/, 'SSVF').
+        gsub(/\bAnd\b/, 'and')
```

Comment:
I hesitate to suggest we use Rails built-in acronym support because historically it's gotten in the way of class names, but I wonder if standardizing this in some way, or at least centralized it for things like **HoH** and **SSVF**.  See `config/initializers/inflections.rb`